server/service/src/sync/translations/location_type.rs

```diff
@@ -0,0 +1,83 @@
+use repository::{ChangelogTableName, ColdStorageTypeRow, StorageConnection, SyncBufferRow};
+use serde::{Deserialize, Serialize};
+
+use super::{PullTranslateResult, SyncTranslation};
+
+#[derive(Deserialize, Serialize)]
+pub struct LegacyLocationTypeRow {
+    #[serde(rename = "ID")]
+    pub id: String,
+    #[serde(rename = "Description")]
+    pub description: String,
+    #[serde(rename = "Temperature_min")]
+    pub temperature_min: f64,
+    #[serde(rename = "Temperature_max")]
+    pub temperature_max: f64,
+}
+// Needs to be added to all_translators()
+#[deny(dead_code)]
+pub(crate) fn boxed() -> Box<dyn SyncTranslation> {
+    Box::new(LocationTypeTranslation)
+}
+
+/// Translates between the legacy LocationTypeRow and the new ColdStorageTypeRow
+pub(super) struct LocationTypeTranslation;
```

Comment:
Hm, comparing to trans_line/invoice_line: https://github.com/msupply-foundation/open-msupply/blob/5259-gaps-sync-translation-for-location_type-cold-storage-type/server/service/src/sync/translations/invoice_line.rs#L93

I think the translator, and the file names should be the OMS name (cold_storage_type), only references to legacy row should be location_type?

Reply:
Updated!